### ISSUE TITLE
Release artifacts (WUTDet dataset, models) on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [MAPGroup/WUTDet#1]
Hi @MAPGroup 🤗

Niels here from the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://hf.co/papers/2604.07759.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw that you are planning to release the **WUTDet** dataset and benchmarks on GitHub. It'd be great to make the checkpoints and dataset available on the 🤗 hub as well, to improve their discoverability and visibility within the computer vision community. We can add tags so that people find them when filtering https://huggingface.co/models and https://huggingface.co/datasets.

## Uploading dataset

WUTDet and the Ship-GEN test set look like very significant contributions for maritime ship detection. Hosting on Hugging Face would allow people to easily explore and load the data:

```python
from datasets import load_dataset

dataset = load_dataset("MAPGroup/WUTDet")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data and the annotated images directly in the browser.

## Uploading models

You've evaluated 20 baseline models across different architectures. Pushing these checkpoints to the Hub would be very valuable for the community to reproduce your benchmarks and use them as pre-trained weights for maritime tasks. See here for a guide: https://huggingface.co/docs/hub/models-uploading.

We encourage researchers to push each model checkpoint to a separate model repository so that download stats work correctly. We can then link the checkpoints directly to the paper page.

Let me know if you're interested or need any help regarding this!

Cheers,

Niels
ML Engineer @ HF 🤗
